datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/instances/openmetrics.yaml

```diff
@@ -120,7 +123,7 @@
     `promhttp_metric_handler_requests`. This submits to Datadog the metric name appended with `.count`.
     For more information, see:
     https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md#suffixes
-    
+
```

Comment:
```suggestion
```

That could be a good idea, but if you add this right here you will also to update ALL the openmetrics integrations in -core and -extras. I don't think it's worth it to be honest

Reply:
We could create a card with all the thing we would like to update in a template and wait until we have enough things to update them all at once

Reply:
oh, my editor did this automatically 👼 

it's a bit silly to keep, but your point about updates is well taken, I'll create a card.